plugins/mcreator-core/jsontriggers/entity_hurt_player.json

```diff
@@ -0,0 +1,42 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "minDamage",
+      "check": "Number"
+    },
+    {
+      "type": "input_value",
+      "name": "maxDamage",
+      "check": "Number"
+    },
+    {
+      "type": "field_data_list_selector",
+      "name": "sourceentity",
+      "datalist": "entity"
+    },
+    {
+      "type": "field_checkbox",
+      "name": "blocked"
+    }
+  ],
+  "inputsInline": true,
+  "previousStatement": null,
+  "nextStatement": null,
+  "colour": 195,
+  "mcreator": {
+    "toolbox_id": "entities",
+    "toolbox_init": [
+      "<value name=\"minDamage\"><block type=\"math_number\"><field name=\"NUM\">0</field></block></value>",
+      "<value name=\"maxDamage\"><block type=\"math_number\"><field name=\"NUM\">5</field></block></value>"
+    ],
+    "inputs": [
+      "minDamage",
+      "maxDamage"
+    ],
+    "fields": [
+      "source_entity",
```

Comment:
You forgot to change to sourceentity here, meaning block in its current state will not work likely

Reply:
i apologize, it should be updated